rules/opentitan/openocd.bzl

```diff
@@ -0,0 +1,20 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+OPENTITANTOOL_OPENOCD_DATA_DEPS = [
+    "//third_party/openocd:jtag_olimex_cfg",
+    "//third_party/openocd:openocd_bin",
+    "//util/openocd/target:lowrisc-earlgrey.cfg",
+    "//util/openocd/target:lowrisc-earlgrey-lc.cfg",
+]
+
+OPENTITANTOOL_OPENOCD_TEST_CMD = """
+    --openocd="$(rootpath //third_party/openocd:openocd_bin)"
+    --openocd-adapter-config="$(rootpath //third_party/openocd:jtag_olimex_cfg)"
+    --openocd-riscv-target-config="$(rootpath //util/openocd/target:lowrisc-earlgrey.cfg)"
+    --openocd-lc-target-config="$(rootpath //util/openocd/target:lowrisc-earlgrey-lc.cfg)"
+    --clear-bitstream
+    --bitstream={bitstream}
```

Comment:
This implies that this config is used for FPGA targets. Should we capture that in the variable name?

I think also from a SiVal perspective, would we use a separate TEST_CMD? In which case it is probably ok to make this one fpga specific.

Reply:
Currently, FPGA targets are the only use-case.  I agree that we should refactor this a bit to work for both FPGA and SiVal targets.

Reply:
Added a note to #19906.